tests/modules/asset_groups/resources/utils.py

```diff
@@ -438,41 +629,6 @@ def commit_asset_group_sighting_sage_identification(
     return response
 
 
-def create_asset_group_with_annotation(
```

Comment:
removed, calling code now creates the group and patches in the dummy annotation as separate steps

Reply:
seems like a good choice to put `create_simple_asset_group` to use in the most places.